### PR TITLE
[FEATURE] Load sequelize classes from api/models

### PR DESCRIPTION
**Pull request by stevenzeiler**
By keeping all sequelize models under api/models we are free to put non-sequelize classes under api/services, which needs to be done to create ValidatorStatsService that returns daily validation stats per validator.

Disables the default ORM called Waterline provided by Sails.js in favor of using our own Sequelize ORM.

### CODE REVIEW

**Comment by stevenzeiler** [comment]
+r

**Comment by wilsonianb** [comment]
I'm guessing `models-path` in `.sequelizerc` needs to be updated

**Comment by stevenzeiler** [comment]
Oh yeah I didn't know we had a .sequelizerc, is that for the sequelize-cli database migrations?

**Comment by wilsonianb** [comment]
I think so

File: api/services/database.js [hunk 1]
```diff
@@ -9,13 +9,16 @@ var sequelize = new Sequelize(process.env.DATABASE_URL, {
...
+
 fs
-  .readdirSync(__dirname)
+  .readdirSync(MODELS_PATH)
   .filter(function(file) {
     return (file.indexOf('.') !== 0) && (file !== basename);
   })
   .forEach(function(file) {
-    var model = sequelize['import'](path.join(__dirname, file));
+    console.log('FILE', file)
```
Comment (wilsonianb) [inline api/services/database.js]: Can this log be removed?

Comment (stevenzeiler) [inline api/services/database.js]: oops yes

**Comment by stevenzeiler** [comment]
The log statement is removed per your request.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**